docs/reference/config/0-app-config.md

```diff
@@ -20,13 +20,15 @@
             - csrf_config
             - debug
             - dependencies
+            - etag
```

Comment:
`etag` was not exposed for some reason so I fixed this while I was looking at the reference doc